.github/workflows/test.yml

```diff
@@ -0,0 +1,102 @@
+---
+# Source: https://github.com/hendrikmaus/rust-workflows
+#
+# Continuous Integration Workflow For Rust
+name: CI
+
+# Define the triggers; usually merges to the repository
+# default branch and pull-requests
+on:
+  push:
+    branches:
+      - main
+      - master
+    paths-ignore:
+      - "docs/**"
+      - "**.md"
+  pull_request:
+    types:
+      - opened
+      - reopened
+      - synchronize
```

Comment:
What does this mean, in practical terms? Does it leave out draft pull requests?

cc @hendrikmaus